include/openthread/dataset.h

```diff
@@ -594,6 +594,18 @@ otError otNetworkNameFromString(otNetworkName *aNetworkName, const char *aNameSt
  */
 otError otDatasetParseTlvs(const otOperationalDatasetTlvs *aDatasetTlvs, otOperationalDataset *aDataset);
 
+/**
+ * This function converts a given Operational Dataset to `otOperationalDatasetTlvs`.
```

Comment:
Can you drop "This function" and just start with "Converts..."?

That way it makes sense in the CLI command reference as well.

Reply:
Changed in new push.